opencog/atomspace/TruthValue.cc

```diff
@@ -94,3 +94,13 @@ bool TruthValue::isDefaultTV() const
     }
     return false;
 }
+
+TruthValueType TruthValue::getType() const
+{
+	return tv_type_;
+}
+
+void TruthValue::setType(TruthValueType t)
+{
+	tv_type_ = t;
```

Comment:
BTW, I added Fuzzy and Probabilistic just a few days ago, for exactly this reason. The merge funcs for fuzzy and probabilistic are completely wrong, and need to be replaced by the correct formulas from the pln book.  The guts of SimpleTV merge func is wrong, as well.   Best to leave CountTV alone for now; if you want to change CountTV, then use either Probabilistic, or create a new TV, say, "FrequencyTruthValue"